README.md

```diff
@@ -2,3 +2,13 @@
 
 # sentiment-analysis
 Simple show case of consuming twitter api and performing sentiment analysis over tweets
+
+### How to contribute
+You need the following dependencies installed:
+- [Gradle](https://gradle.org/)
+- [ElasticSearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/_installation.html)
```

Comment:
can we add this for Windows users (https://www.elastic.co/downloads/elasticsearch)

Reply:
Sure, will do it.